### PR TITLE
Feature/2489/Implement custom view descriptions

### PR DESCRIPTION
**Pull request by Hall-Ma**
# Implement Custom View Descriptions

Issue: #2489

## Description

- add description to every custom view:
   - name
   - area metric
   - height metric
   - edge metric (if available)
   - color schema
- change value of `MULTIPLE` mode to `STANDARD`  in `CustomConfigFileSelectionMode` (support configs with old property value)

## Screenshots or gifs

image

### CODE REVIEW

File: visualization/app/codeCharta/ui/customConfigs/customConfigList/customConfigItemGroup/customConfigDescription/customConfigDescription.component.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div class="config-description-container">
+	<p><b>Name: </b>{{ customConfigItem.name }}</p>
+	<p><b>Selection Mode:</b> {{ customConfigItem.mapSelectionMode | titlecase }}</p>
+	<p><i class="fa fa-arrows-alt"></i> {{ customConfigItem.metrics.areaMetric }}</p>
+	<p><i class="fa fa-arrows-v"></i> {{ customConfigItem.metrics.heightMetric }}</p>
```
Comment (ce-bo) [inline visualization/app/codeCharta/ui/customConfigs/customConfigList/customConfigItemGroup/customConfigDescription/customConfigDescription.component.html:5]: What about the selected color and edge metric?
I would suggest to just display them even if they are the same.

**Comment by BridgeAR** [comment]
This already looked really great!
To simplify the view a bit further, I suggest three changes as outlined in the picture:

Simplified

**Comment by Hall-Ma** [comment]
image
@ce-bo Should we drop the "Standard Mode" label, as Ruben suggested?

**Review by RomanenkoVladimir — APPROVED** [review]
LGTM, when the test is fixed

**Review by LuisGuenther — APPROVED** [review]
LGTM too

**Comment by ce-bo** [comment]
The UI now looks very nice! Do you show the color metric?

**Pull request merged**